src/Microsoft.ML.Core/Utilities/Utils.cs

```diff
@@ -615,37 +615,75 @@ public static string ExtractLettersAndNumbers(string value)
             return Regex.Replace(value, "[^A-Za-z0-9]", "");
         }
 
-        public static bool IsSorted(IList<float> values)
+        /// <summary>
+        /// Checks that an input IList is monotonically increasing.
+        /// </summary>
+        /// <param name="values">An array of values</param>
+        /// <returns>True if the array is monotonically increasing; false otherwise.
```

Comment:
>if the array is monotonically increasing [](start = 26, length = 40)

maybe:
`if each element is greater or equal compare to previous elements` for simple minded people?

Reply:
Good call. I added your language to the xml docs :)